workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -591,8 +591,19 @@ oneOf:
     artifacts:
       type: object
       title: Artifacts
-      description: >-
+      description: |-
         Artifact upload map example: ```{"public/build.tar.gz": {"path": "/home/worker/build.tar.gz", "expires": "2016-05-28T16:12:56.693817Z", "type": "file"}}```
+        Artifacts can be an individual `file`, a `directory` containing
+        potentially multiple files with recursively included subdirectories,
+        or a `volume` (d2g only) which will create a volume mount from the
```

Comment:
Since this section of the schema is for d2g only, i think you can safely remove `(d2g only)`. Or you can write `(not supported in the original docker worker implementation)`. But probably just removing the brackets is the least confusing, I would say.